recipe-server/normandy/base/utils.py

```diff
@@ -70,7 +70,7 @@ def filter_m2m(qs, field, values):
     return qs
 
 
-def sri_hash(data):
+def sri_hash(data, url_safe=False):
```

Comment:
Would calling this a "SRI Hash" cause confusion later on? This is simply a hash, and we're not exactly in the context of subresource integrity here.